lib/measures/tbd/topolys.rb

```diff
@@ -0,0 +1,4 @@
+require_relative "resources/topolys/version"
```

Comment:
Similar to lib/measures/tbd/tbd.rb, assembling the measure from static Topolys files.